recipes/quickfast/all/conanfile.py

```diff
@@ -0,0 +1,208 @@
+        elif self.settings.os == 'Macos':
+            return 'macosx'
+        elif self.settings.os == 'Linux':
+            return 'linux'
+        elif self.settings.os == 'Windows':
+            return self._wss_platform
+        else:
+            raise ConanInvalidConfiguration("QuickFAST cannot be built for the {} operating system"
+                                            .format(self.settings.os))
+
+    @property
+    def _visual_studio_build_type(self):
+        compiler = self.settings.compiler
+        version = Version(compiler.version.value)
+
+        if "8" <= version <= "14":
+            return "v" + version
+        elif version == "14":
+            return "vs2017"
+        elif version == "16":
+            return "vs2019"
+        else:
+            raise ConanInvalidConfiguration("QuickFAST cannot be built with Visual Studio version {}"
+                                            .format(version))
+
+    @property
+    def _build_type(self):
+        if self.settings.compiler == "Visual Studio":
+            return self._visual_studio_build_type
+        else:
+            return 'make'
+
+    @property
+    def _core_value_template_linkflags(self):
+        if self.settings.os == "Macos":
+            return ' -value_template "linkflags+=-framework CoreServices -framework CoreFoundation"'
+        return ''
+
+    @property
+    def _mwc_command_line(self):
+        link_flag = ' -static' if not self.options.shared else ' '
+        fpic_flag = '-fPIC' if self.options.get_safe("fPIC", default=True) else ''
+        platform = self._platform
+        build_type = self._build_type
+        core_value_template_linkflags = self._core_value_template_linkflags
+
+        return 'perl -S mwc.pl -type {} -exclude src/Examples'.format(build_type) + \
+               link_flag + \
+               ' -value_template "pic={}"'.format(fpic_flag) + \
+               ' -value_template platforms={}'.format(platform) + \
+               core_value_template_linkflags + \
+               ' -value_template extracppflags+=-DBOOST_BIND_GLOBAL_PLACEHOLDERS' + \
+               ' QuickFAST.mwc'
+
+    def _configure_msbuild(self):
+        if self._msbuild:
+            return self._msbuild
+
+        self.run(self._mwc_command_line)
+        self._msbuild = MSBuild(self)
+        return self._msbuild
+
+    def _configure_autotools(self):
+        if self._env_build:
+            return self._env_build, self._args
+
+        self._env_build = AutoToolsBuildEnvironment(self, win_bash=self.settings.os is "Windows")
+        self._args = ['CONAN_MAKE_FILE=' + os.path.join(self.build_folder, "conanbuildinfo.mak")]
+        self.run(self._mwc_command_line, win_bash=self.settings.os is "Windows")
+        return self._env_build, self._args
+
+    def _patch_sources(self):
+        # Patch taken from:
+        # https://raw.githubusercontent.com/microsoft/vcpkg/master/ports/quickfast/00001-fix-boost-asio.patch
```

Comment:
```suggestion
```

move the comment the patch in conandata.yml. For future versions it will be invalid here.